fiftyone/server/mutation.py

```diff
@@ -322,7 +322,7 @@ async def delete_saved_view(
         # If the current view is deleted, set the view state to the full
         # dataset view
         state = get_state()
-        if view_name and state.view_name == view_name:
+        if view_name and state.view and state.view.name == view_name:
```

Comment:
`state.view` is expensive because `DatasetView.__bool__` computes `len(view) > 0`. I think you should explicitly use `state.view is not None` instead.

Reply:
good to know 👍

Reply:
👍 I defer to an App code owner for approval from here